cli/src/commands/commit.rs

```diff
@@ -131,14 +132,24 @@ new working-copy commit.
     }
 
     let description = if !args.message_paragraphs.is_empty() {
-        join_message_paragraphs(&args.message_paragraphs)
+        let mut description = join_message_paragraphs(&args.message_paragraphs);
+        if !description.is_empty() {
+            commit_builder.set_description(description);
+            let temp_commit = commit_builder.write_hidden()?;
+            description = add_trailers(ui, &tx, &temp_commit)?;
```

Comment:
nit: perhaps, we can apply the same optimization as `jj describe`.

```rust
if let Some(trailer_template) = parse_trailers_template(ui, &tx)? {
    let temp_commit = commit_builder.write_hidden()?;
    ...
}
```

(or do that in `add_trailers(ui, &tx, &mut commit_builder)`?)